packages/cspell-tools/package.json

```diff
@@ -38,22 +38,28 @@
   },
   "homepage": "https://github.com/streetsidesoftware/cspell#readme",
   "dependencies": {
+    "@types/node": "^14.14.6",
```

Comment:
same as above

Reply:
I'm only seeing it internal now, so I'll swap back to devDependency